scripts/ci-e2e.sh

```diff
@@ -84,6 +84,24 @@ fi
 
 "hack/ci/create_devstack.sh"
 
+# TODO: Upload image to swift for e2e clusterctl upgrade tests
+# This is just a quick hack to test the process
+make e2e-image
+make hack/tools/bin/gojq
+docker save -o ${ARTIFACTS}/capo-e2e-image.tar gcr.io/k8s-staging-capi-openstack/capi-openstack-controller:e2e
+export OS_CLOUD=capo-e2e
+VIRTUAL_ENV_DISABLE_PROMPT=1 source /tmp/openstack-venv/bin/activate
+openstack container create --public capo-e2e
+openstack object create capo-e2e ${ARTIFACTS}/capo-e2e-image.tar --name capo-e2e-image.tar
+ACCOUNT=$(openstack object show capo-e2e capo-e2e-image.tar -f json | hack/tools/bin/gojq -r .account)
```

Comment:
I am not against swift, so double check here whether we have other simpler way?
e.g glance can store files , httpd(apache) can provide data through wget .. 
so just a question here

Reply:
Thank you for this! I was not aware of the httpd server running on the controller. This was indeed much easier to use.